mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/documentdialog/DocumentDialogPresenter.java

```diff
@@ -58,13 +60,25 @@ public void onCompleted() {
                     @Override
                     public void onError(Throwable e) {
                         getMvpView().showProgressbar(false);
-                        getMvpView().showError(R.string.failed_to_upload_document);
+                        if (e instanceof HttpException) {
```

Comment:
@tarun0 why don't you use https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/java/com/mifos/utils/MFErrorParser.java#L25